source/ui/beamline/AMXRFDetailedDetectorView.h

```diff
@@ -48,13 +48,6 @@ class AMXRFDetailedDetectorView : public AMXRFBaseDetectorView
 	/// Re-implementing but still going to use the base class buildDetectorView since this view is merely adding to it.
 	virtual void buildDetectorView();
 
-	/// Returns the energy range used for filtering.  If no range has been provided the range is null.
-	const AMRange &energyRange() const { return emissionLineValidator_->range(); }
```

Comment:
Yes they were supposed to be removed in the initial refactor, as I went over all the functions again I noticed they were defined in both AMSpectrumAndPeriodicTableView and here.